pkg/binary/envoy/debug/testdata/log.yaml

```diff
@@ -59,4 +59,4 @@ admin:
   address:
     socket_address:
       address: 0.0.0.0
-      port_value: 15000
+      port_value: 0
```

Comment:
this was a revelation, the insight that you can set this to zero and use --admin-address-path to avoid constant port conflicts!